<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>img1</class>
 <widget class="QWidget" name="img1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imgshow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="img_process">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>img_process</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="img_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>600</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imgshow_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>img_name</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgshow_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>algorithm_name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="algorithm_name">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>600</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="param1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>600</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="param2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
